updates_packagekit_kde-kde-screenlock-enabled-20161009.json

```diff
@@ -0,0 +1,16 @@
+{
+  "tags": [
+    "kde-disable-screenlock",
+    "kde-screenlock-enabled"

```

Comment:
Of course it is pretty unlikely that someone will try to match on kde-* needles in a non-kde environment but I guess you should say ENV-DESKTOP-kde